### DISCUSSION TITLE
Neue Fusion DTU und Konfig-Probleme.

### DISCUSSION THREAD

**Discussion by Gururom** [hoylabs/OpenDTU-OnBattery#1184] [General]
Hallo,
habe mir wegen der 4MB Sache eine neue DTU gekauft von Allianceapps, komme mit der Konfig jedoch nicht klar.
Habe bereits eine openDTUonBattery am laufen, also will diese nur ersetzen.
Bei der Stromzählereinstellung habe ich HTTPs und JSON eingestellt, kann jedoch keinen Pfad zu meinem Shelly eintragen, da dieses Feld nicht da ist???
![Stromzähler1](https://github.com/user-attachments/assets/4674de74-eced-4d9b-9a32-41d910fcba04)
![Stromzähler2](https://github.com/user-attachments/assets/893fe269-c5c7-447d-9903-a969ae3f27eb)
wie komm ich zum Teufel dazu????

Zweites Problem was sich mir stellt.
Will die Firmware neu aufspielen, vielleicht liegts daran, aber welche nehme ich für das Fusion Modell???
![Firmeware](https://github.com/user-attachments/assets/1fa0609e-bd6d-426c-bb90-297c0bd4edeb)

**Comment by spcqike** [comment]
ich vermute das ist ein Bug in der Version, mit der das Modul ausgeliefert wurde. Dieser kam auch hier im PR auf, wurde aber eigentlich behoben
https://github.com/helgeerbe/OpenDTU-OnBattery/pull/1077#issuecomment-2200817594

auch die PIO Umgebung "opendtufusionv2" zeigt, dass es keine offizielle firmware aus dem Repo ist.

für ein Fusion Board brauchst du eigentlich die ``opendtu-onbattery-generic_esp32s3_usb``. installiere dir mal die factory Version (``
opendtu-onbattery-generic_esp32s3_usb.factory.bin``) per USB neu, ab dann halt per OTA die non-factory Versionen

**Comment by schlimmchen** [comment]
> ich vermute das ist ein Bug in der Version, mit der das Modul ausgeliefert wurde.

@Gururom Ich nehme stark an, dass deine OpenDTU Fusion **ohne** Software ausgeliefert wurde. Ist das korrekt?

> auch die PIO Umgebung "opendtufusionv2" zeigt, dass es keine offizielle firmware aus dem Repo ist.

@spcqike Das kannst du so nicht sagen. Die opendtufusionv2 Platform IO Umgebung ist eine gültige PIO Umgebung. Sie entspricht der genereic_esp32s3_usb, definiert aber die NRF24 und CMT2300 Pins in der Firmware. Sie ist in der Tat keine Variante der in Releases enthaltenen Firmwares.
 
@Gururom Ich unterstelle, dass du Firmware selbst gebaut hast und du hast versäumt die webapp vorher zu bauen. https://opendtu-onbattery.net/firmware/compile_webapp/ denn dann hast du genau dieses Fehlerbild.

**Reply by Gururom** [reply]
nein, habe die so von der Allianceapp geliefert bekommen, habe extra gefragt ob ich sie mit openDTUonBattery bekommen kann.
Also ja, hab sie so geliefert bekommen.
Habs jetzt versucht zu flashen, aber es ist mir total misslungen, hoffe das mir der Verkäufer nochmal richtet.
Mit dem flashen über USB hatte ich eh Bauschmerzen.
Naja, nun muss ich sehen.

**Reply by schlimmchen** [reply]
Das ist ja Mist... Ich versuche Tim eine passende Nachricht zuzustecken.

> Habs jetzt versucht zu flashen, aber es ist mir total misslungen, hoffe das mir der Verkäufer nochmal richtet.

Man wird dir sicher helfen. Allerdings kannst du dir garantiert auch selbst helfen um Zeit zu sparen. Schau mal hier: https://opendtu-onbattery.net/firmware/webinstall/ Magst du das mal ausprobieren? Damit haben viele Benutzer gute Erfahrungen gemacht.

**Reply by schlimmchen** [reply]
@cat24max Schau mal bitte hier vorbei um zu vermeiden, "schlechte" Firmware mit den Fusions auszuliefern.

**Reply by spcqike** [reply]
> opendtufusionv2 Platform IO Umgebung ist eine gültige PIO Umgebung.

ok das war mir nicht bekannt. Und wenn sie nicht in den releases enthalten ist, weiß ich auch warum ich sie nicht kenne :)

**Reply by Gururom** [reply]
> Das ist ja Mist... Ich versuche Tim eine passende Nachricht zuzustecken.
> 
> > Habs jetzt versucht zu flashen, aber es ist mir total misslungen, hoffe das mir der Verkäufer nochmal richtet.
> 
> Man wird dir sicher helfen. Allerdings kannst du dir garantiert auch selbst helfen um Zeit zu sparen. Schau mal hier: https://opendtu-onbattery.net/firmware/webinstall/ Magst du das mal ausprobieren? Damit haben viele Benutzer gute Erfahrungen gemacht.

Hab da mal kurz reingeschaut, ich teste das heut abend mal, bin aber nicht so sicher das ich das hinbekomme.
Na mal schaun

**Reply by walter2323** [reply]
Ich hänge mich mal an.
Habe genau die gleichen Schwierigkeiten, aktuell läuft das System aber Update via USB scheitert...

**Reply by Gururom** [reply]
Versuchs doch mal mit dem Webflasher, scheint ganz gut und auch selbsterklärend zu sein

**Reply by schlimmchen** [reply]
@walter2323 Lies bitte nochmal aufmerksam [die Doku](https://opendtu-onbattery.net/firmware/flash_esp), da ist zuletzt noch einiges hinzugekommen um es hoffentlich wirklich klar zu machen, wie man sich im Zweifel helfen kann.

**Reply by walter2323** [reply]
Vielen lieben Dank für die Hilfe!
Heute hat es geklappt, mir war nicht klar gewesen, dass ich auf dem Board die entsprechenden Knöpfe drücken muss. (Boot/Reset)

**Comment by Gururom** [comment]
So, mein Problem ist gelöst, nochmal vielen Dank an allianceapp für den promten Service.
War wohl wirklich ein defektes/unpassendes Image auf der fusion.
Habe eine neue bekommen mit der aktuellen Firmware und 16mb Speicher.
Top
